sys/include/congure/quic.h

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (C) 2021 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_congure_quic    CongURE implementation of QUIC's CC
+ * @ingroup     sys_congure
+ * @brief       Implementation of QUIC's congestion control algorithm for the
+ *              CongURE framework.
+ * @{
+ *
+ * @file
+ *
+ * @author  Martine S. Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef CONGURE_QUIC_H
+#define CONGURE_QUIC_H
+
+#include "ztimer.h"
+
+#include "congure.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Constants for the congestion control.
+ *
+ * Example usage:
+ *
+ * ~~~~~~~~~~~~~~~~ {.c}
+ * static const congure_quic_snd_consts_t consts = {
+ *     .cong_event_cb = _maybe_send_one_pkt,
+ *     .init_wnd = 12000,
+ *     .min_wnd = 2400,
```

Comment:
I just want to know the source of these variables.
Are they from the specification (then link to it), did you draw them from a hat?

Reply:
Done.